crypto-intermediary/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "token-association-js-sdk",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "type": "module",
+  "dependencies": {
+    "@hashgraph/sdk": "^2.31.0",
```

Comment:
Let's update the SDK version to the latest (v2.44 as of April).

Note that this may require small changes to the `bob.js` and `alice.js` files.